src/Plugin/EloquaAppCloudContentResponder/DebugResponderAsync.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Drupal\eloqua_app_cloud\Plugin\EloquaAppCloudContentResponder;
+
+use Drupal\eloqua_app_cloud\Plugin\EloquaAppCloudContentResponderBase;
+use Drupal\eloqua_app_cloud\Plugin\EloquaAppCloudContentResponderInterface;
+use Drupal\eloqua_rest_api\Factory\ClientFactory;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * @EloquaAppCloudContentResponder(
+ *  id = "ContentDebugResponderAsync",
+ *  label = @Translation("Asynchronous Content Debug Responder"),
+ *  description = "Simple asynchronous content debugging tool that always returns the same HTML for every record",
+ *  respond = "asynchronous",
+ *  api = "contacts",
+ *  queueWorker = "eloqua_app_cloud_content_queue_worker",
+ *  fieldList = {
+ *    "EmailAddress" = "{{Contact.Field(C_EmailAddress)}}"
+ *   },
+ *   requiresConfiguration = "false",
+ *   height = "",
+ *   width = "",
+ *   editorImageUrl = ""
```

Comment:
Are these actually defined/used anywhere?  re: `height`, `width`, `editorImageUrl`

Reply:
Eloqua requires them and fails silently if they are not returned. I figured I might as well put them in the annotation for clarity. Will add a comment to that effect.